src/main/resources/db/migration/common/V1_2__prison_supported.sql

```diff
@@ -0,0 +1,10 @@
+CREATE TABLE RISK_PROFILER.PRISON_SUPPORTED
+(
+    PRISON_ID       VARCHAR(6) PRIMARY KEY,
+    START_DATE_TIME TIMESTAMP NOT NULL
+);
+
+COMMENT ON TABLE RISK_PROFILER.PRISON_SUPPORTED IS 'Records the prisons which have started using risk profiler';
+
+COMMENT ON COLUMN RISK_PROFILER.PRISON_SUPPORTED.PRISON_ID IS 'Prison ID (NOMIS column is Agency ID)';
+COMMENT ON COLUMN RISK_PROFILER.PRISON_SUPPORTED.START_DATE_TIME IS 'Indicates when the prison started using risk profiler';
```

Comment:
editing migration scripts  - are you going to drop the database on dev so they are reapplied?

Reply:
Yes indeed, Ill need to do this first otherwise the flyway migration will fail